metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -1114,29 +1165,23 @@ def get_connection_objects(
         filter_pages = get_filter_pages(query_filter, page_size)
 
         for filter_page in filter_pages:
-            total_count = page_size
             has_next_page = 1
-            offset = 0
+            current_cursor: Optional[str] = None
```

Comment:
how do cursors and page sizes interact? why are we using cursors in some places and offsets in others? overall - this code is pretty difficult to understand

Reply:
Could you please point to a code where are we using offset. As per my knowledge I updated the code to use cursor only.

How they interact is mentioned in documentation of `get_connection_object_page`

Reply:
ah I see